### PR TITLE
{data}[intel/2015b] HDF5 1.8.16 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/731c31ffb5430724292c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/fdb1e0d1ba12cc357d37 for a full test report.

File: easybuild/easyconfigs/h/HDF5/HDF5-1.8.16-intel-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name = 'HDF5'
+version = '1.8.16'
+
+homepage = 'http://www.hdfgroup.org/HDF5/'
+description = """HDF5 is a unique technology suite that makes possible the management of
+ extremely large and complex data collections."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+toolchainopts = {'optarch': True, 'pic': True, 'usempi': True}
```
Comment (wpoely86) [inline easybuild/easyconfigs/h/HDF5/HDF5-1.8.16-intel-2015b.eb]: also add one without `mpi`

**Comment by wpoely86** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/a136e5e79c00ada4fba3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ae5ae88f62c2fe9fc136 for a full test report.

File: easybuild/easyconfigs/h/HDF5/HDF5-1.8.16-intel-2015b.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/h/HDF5/HDF5-1.8.16-intel-2015b.eb]: done, thx

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**